aiida/cmdline/utils/daemon.py

```diff
@@ -73,4 +73,31 @@ def get_daemon_status(client):
     template = ('Daemon is running as PID {pid} since {time}\nActive workers [{nworkers}]:\n{workers}\n'
                 'Use verdi daemon [incr | decr] [num] to increase / decrease the amount of workers')
 
-    return template.format(**info)
\ No newline at end of file
+    return template.format(**info)
+
+
+def print_last_process_state_change(process_type='calculation'):
+    """
+    Print the last time that a process of the specified type has changed its state.
+    This function will also print a warning if the daemon is not running.
+
+    :param process_type: the process type for which to get the latest state change timestamp.
+        Valid process types are either 'calculation' or 'work'.
+    """
+    from aiida.cmdline.utils.echo import echo_info, echo_warning
+    from aiida.daemon.client import DaemonClient
+    from aiida.utils import timezone
+    from aiida.common.utils import str_timedelta
+    from aiida.work.utils import get_process_state_change_timestamp
+
+    client = DaemonClient()
+
+    timestamp = get_process_state_change_timestamp(process_type)
```

Comment:
One should return a different string if this returns `None` (before it was writing "Never"). I think this would now crash the first time one runs AiiDA.